### DISCUSSION TITLE
Connection to Stentura 8000

### DISCUSSION THREAD

**Discussion by joyholaday** [opensteno/plover#1591] [Q&A and Support] [answered]
I am attermpting to connect a Stentura 8000 to a Windows 10 machine with a Keyspan USA-19HS Serial Adapter, but when I try to select the created COM port in the configure dialog box, BOTH options for Flow Control are unchecked and the checkboxes are disabled.  It seems that for this using RTS/CTS should be checked?

**Comment by mommyjimco** [comment]
I am also trying to connect with a Stentura 8000 to a Windows 10. Did you ever get an answer for this?

**Comment by joyholaday** [comment]
Still working on it but got LOTS of help.  It seems the Stentura 8000 requires a null modem cable (or the original cable that came with the unit).  The settings on the COM port should be 9600, 8, N, 1, no flow control.  Or at least that's what we think will work.  I will update once my null modem adapter comes in and I give that a shot.  Pretty confident that it is going to work, though.
Respectfully.Don Ryan (for joyholaday - the real stenographer, I'm just tech support!)
    On Wednesday, March 1, 2023 at 05:52:19 PM MST, mommyjimco ***@***.***> wrote:  
 
 


I am also trying to connect with a Stentura 8000 to a Windows 10. Did you ever get an answer for this?

**Comment by joyholaday** [comment]
It works!!!  Plugged-in the serial cable adding the null modem adapter, no connect, checked the Print Direct setting and found it to be off, toggled it to on, no connect, then (I know, kinda jumbled set of steps), set speed to 2400 baud,no connect,  then turned direct print off, and it worked.  I am going to go back and see if I can get it to work at 9600 baud since that is the default.
THANK YOU VERY MUCH FOR THE HELP!!!

**Comment by joyholaday** [comment]
No.  It does not like 9600 baud in its present configuration - perhaps it is possible to change that, but it seems like it would not be necessary - don't think even my wife, as fast as she is, could outrun it.

**Comment by joyholaday** [comment, marked as answer]
So, ...to summarize, I am using Plover 4.0.0.dev12 on Windows 10, a Stentura 8000 writer, the defaults for a stentura machine with the exception of changing the speed to 2400 baud, using this cable:
OIKWAN USB to RS232, USB Serial Adapter with FTDI Chipset,USB 2.0 to Male DB9 Serial Cable for Windows 10, 8, 7, Vista, XP, 2000, Linux and Mac OS(6ft)…
and this null modem adapter:
DB9 null modem male to female slimline data transfer serial port adapter 2 Pack
(only one of them, of course) so the settings are 2400 baud, 8 data bits, 1 stop bit, No Parity, No Flow Control, and a timeout of 2(units?).
Direct Print is OFF on the writer....and it works!
